### PR TITLE
V1.1.2 fix 

### PR DESCRIPTION
**Pull request by Feroz-Shaikh**
This is already fixed on v1.1.3 release. However providing a quick fix for https://github.com/tmobile/jazz/issues/63.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 2) Run the below command to run the installation wizard.
 
 ```
-curl -L https://raw.githubusercontent.com/tmobile/jazz-installer/v1.1.2/installscripts/terraforminstaller/rhel7Installer.sh -o rhel7Installer.sh && chmod +x rhel7Installer.sh && ./rhel7Installer.sh v1.1.2 && cd ./jazz-installer/installscripts/wizard && ./run.py 
+curl -L https://raw.githubusercontent.com/tmobile/jazz-installer/master/installscripts/terraforminstaller/rhel7Installer.sh -o rhel7Installer.sh && chmod +x rhel7Installer.sh && ./rhel7Installer.sh master && cd ./jazz-installer/installscripts/wizard && ./run.py
```
Comment (suryajak) [inline README.md]: Please do a pull from master as your local repo seems to be out of sync. We have updated content and we no longer have install instructions in README.

File: installscripts/wizard/run.py [hunk 2]
```diff
@@ -43,11 +43,9 @@ def check_jenkins_user(url, username, passwd):
 
     if 'authenticated' in open('output').read():
         os.remove('output')
-        os.remove('jenkins-cli.jar')
```
Comment (suryajak) [inline installscripts/wizard/run.py]: Once you rebase with master, please confirm that this is the only change that is needed.

**Comment by Feroz-Shaikh** [comment]
@suryajak I can confirm that this is the only change we need to fix this. 
Moreover I have not created the branch from master, instead I checked out the last stable tag v1.1.2 and created the branch out of this tag. The readme file still exists in tagged version: https://github.com/tmobile/jazz-installer/tree/v1.1.2. I believe you might have done a master to master push to remove readme.
Please let me know if I should create a new branch from master and generate a PR again?

**Comment by suryajak** [comment]
@Feroz-Shaikh: Your PR is for master, so you should checkout master branch and then apply the fix.

**Comment by Feroz-Shaikh** [comment]
Closing this PR in favor of https://github.com/tmobile/jazz-installer/pull/27

**Pull request closed without merging**